### PR TITLE
PREAPPS-7557: 'Address in' Filter support in Modern UI

### PR DESCRIPTION
**Pull request by namita-uttarkar-telus**
Added type attribute to addressBook API call as the backend API accepts type attribute.

### CODE REVIEW

**Review by miteshsavani810 — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

File: src/schema/schema.graphql [hunk 1]
```diff
@@ -2718,6 +2718,7 @@ input HeaderCheckConditionInput {
 	header: String!
 	index: Int
 	negative: Boolean
+	type: String
```
Comment (silentsakky) [inline src/schema/schema.graphql:2721]: I don't see this parameter present in soap api https://files.zimbra.com/docs/soap_api/10.0.0/api-reference/zimbraMail/ModifyFilterRules.html
is this gap in documentation or we are doing something custom

Comment (namita-uttarkar-telus) [reply src/schema/schema.graphql:2721]: I have referred classic UI for ModifyFilterRulesRequest API. Type parameter is sent within addressBookTest in the condition when fourth filter drop down value in 'Address in' filter is 'Contact'. So have added type parameter. 
```
{
    "condition": "anyof",
    "addressBookTest": [
        {
            "header": "FROM",
            "type": "contacts"
        }
    ]
}
```
Just checked by removing type parameter, API call got successfully executed. 
I will be closing this branch.

**Comment by namita-uttarkar-telus** [comment]
Closing this branch, as type parameter is not required for addressBookTest

**Comment by namita-uttarkar-telus** [comment]
Resorting and reopening this branch just for the circle-ci test cases to get executed. Once https://github.com/ZimbraOS/zm-x-web/pull/4697 gets closed I will close and delete this branch.
Note: There are zero file changes in this branch.

**Pull request closed without merging**